Client/Frontend/Sync/BraveCore/BraveSyncAPI+Utilities.swift

```diff
@@ -38,6 +40,23 @@ extension BraveSyncAPI {
         Preferences.Chromium.syncEnabled.value = false
     }
     
+    func enableSyncTypes() {
+        guard let syncProfileService =
+                (UIApplication.shared.delegate as? AppDelegate)?.braveCore?.syncProfileService else {
+            return
+        }
```

Comment:
As mentioned in other places, this is not a good solution

Reply:
And same as https://github.com/brave/brave-ios/pull/3703#discussion_r668082916, evry bookmark and history API usage will be addressed.